_data/guides.yaml

```diff
@@ -31,6 +31,9 @@ categories:
       - title: Securing WildFly Apps on OpenShift with OpenID Connect Using Additional Scope Values
         url: /guides/security-oidc-scope-openshift
         description: Learn how to secure applications deployed to WildFly on OpenShift with OpenID Connect using additional scope values.
+      - title: Sending Request Objects as A JWT Using Request Parameters for OIDC
```

Comment:
Just to be consistent with other titles, would be good to use "OpenID Connect" here instead of OIDC.

Reply:
fixed! 
Thanks @fjuma